docs/src/modules/components/AppNavDrawerItem.tsx

```diff
@@ -256,10 +262,14 @@ export const sxChip = (color) => [
     letterSpacing: '.04rem',
     height: '16px',
     border: 1,
+    // @ts-ignore
```

Comment:
Is there something we can do to fix the ts ignores?

Reply:
I updated